Healthy/Classes/Utilities/Authentication/Login/FacebookAuthenticator.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import UIKit
+import FBSDKLoginKit
+
+// Provide AppID
+class FacebookAuthenticator: NSObject {
```

Comment:
You need to conform authenticator and move all the needed logic inside authenticate. Check Google authenticator please.